protostar/commands/test/starkware/execution_resources_summary_test.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+
+from protostar.commands.test.starkware.execution_resources_summary import (
+    CountStatistic,
+    CountSeriesStatistic,
+    Statistic,
+    ExecutionResourcesSummary,
+)
```

Comment:
If we reimplement `CountSeriesStatistic` with and append we should check following properties for  `CountSeriesStatistic` + `CountSeriesStatistic` -> `CountSeriesStatistic` addition :

- `CountSeriesStatistic([1,2,3]) + CountSeriesStatistic([]) == CountSeriesStatistic([1,2,3])`
- `(CountSeriesStatistic([1]) + CountSeriesStatistic([2])) + CountSeriesStatistic([3]) == CountSeriesStatistic([1]) + (CountSeriesStatistic([2]) + CountSeriesStatistic([3]))`